docs/source/quicktour.mdx

```diff
@@ -14,186 +14,186 @@ specific language governing permissions and limitations under the License.
 
-
-```py
->>> classifier('We are very happy to show you the 🤗 Transformers library.')
-[{'label': 'POSITIVE', 'score': 0.9998}]
+```python
+>>> classifier("We are very happy to show you the 🤗 Transformers library."")
+[{"label": "POSITIVE", "score": 0.9998}]
 ```
 
-That's encouraging! You can use it on a list of sentences, which will be preprocessed then fed to the model, returning
-a list of dictionaries like this one:
+For more than one sentence, pass a list of sentences to the [`pipeline`] which returns a list of dictionaries:
 
-```py
+```python
 >>> results = classifier(["We are very happy to show you the 🤗 Transformers library.",
 ...            "We hope you don't hate it."])
 >>> for result in results:
-...     print(f"label: {result['label']}, with score: {round(result['score'], 4)}")
+...     print(f"label: {result["label"]}, with score: {round(result["score"], 4)}")
 label: POSITIVE, with score: 0.9998
 label: NEGATIVE, with score: 0.5309
 ```
 
-To use with a large dataset, look at [iterating over a pipeline](main_classes/pipelines)
+The [`pipeline`] can also iterate over an entire dataset. Start by installing the 🤗 Datasets library:
+
+```bash
+pip install datasets 
+```
+
+Create a [`pipeline`] with the task you want to solve for, and the model you want to use:
 
-You can see the second sentence has been classified as negative (it needs to be positive or negative) but its score is
-fairly neutral.
+```python
+>>> from transformers import pipeline
+>>> pipe = pipeline("automatic-speech-recognition", model="facebook/wav2vec2-base-960h", device=0)
+```
 
-By default, the model downloaded for this pipeline is called "distilbert-base-uncased-finetuned-sst-2-english". We can
-look at its [model page](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) to get more
-information about it. It uses the [DistilBERT architecture](model_doc/distilbert) and has been fine-tuned on a
-dataset called SST-2 for the sentiment analysis task.
+Load the dataset you want to iterate over:
 
-Let's say we want to use another model; for instance, one that has been trained on French data. We can search through
-the [model hub](https://huggingface.co/models) that gathers models pretrained on a lot of data by research labs, but
-also community models (usually fine-tuned versions of those big models on a specific dataset). Applying the tags
-"French" and "text-classification" gives back a suggestion "nlptown/bert-base-multilingual-uncased-sentiment". Let's
-see how we can use it.
+```python
+>>> import datasets
+>>> dataset = datasets.load_dataset("superb", name="asr", split="test")
+```
 
-You can directly pass the name of the model to use to [`pipeline`]:
+Now iterate over the dataset with the pipeline. This should be just as fast as running a custom loop on a GPU.
```

Comment:
GPU support has not been introduced at this stage, and a random user running this code probably won't have one.